### ISSUE TITLE
RCS1190 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#387]
The first line is flagged by RCS1190 but not the second. Is this a false negative?

```csharp
namespace RCS1190_False_Negative
{
    public class Class
    {
        private static void Method()
        {
            string test1 = "a" + "b" + 1;
            string test2 = 1 + "a" + "b";
        }
    }
}
```
